Civi/API/Kernel.php

```diff
@@ -220,6 +220,9 @@ public function authorize($apiProvider, $apiRequest) {
     /** @var \Civi\API\Event\AuthorizeEvent $event */
     $event = $this->dispatcher->dispatch('civi.api.authorize', new AuthorizeEvent($apiProvider, $apiRequest, $this, \CRM_Core_Session::getLoggedInContactID() ?: 0));
     if (!$event->isAuthorized()) {
+      // We log for visibility and throw an exception to allow the parent to handle the error
+      \Civi::log()->error('API Request Authorization failed');
+      CRM_Core_Error::backtrace('backTrace', TRUE);
```

Comment:
Does this need the backslash at the front? Also maybe instead of logging two entries each time, just put the text instead of the uninteresting 'backTrace', i.e. just one line:

`\CRM_Core_Error::backtrace('API Request Authorization failed', TRUE);`

Reply:
Agree with @demeritcowboy's suggestion about one call instead of 2. Test failures say that yes it does need the backslash prefix.

Reply:
oops yep, thanks, I've updated the PR